### DISCUSSION TITLE
Problem to build the panel

### DISCUSSION THREAD

**Discussion by MinePlay85** [pterodactyl/panel#3074] [Support] [answered]
Hello,
i trying to build the panel because I want to add addon but it gives me an error with yarn.
![F076439F-16A5-4259-BE65-56D84F467821](https://user-images.githubusercontent.com/69992311/107127479-aa928e00-68b6-11eb-927d-20153ed07050.jpeg)

**Comment by matthewpi** [comment, marked as answer]
![](https://user-images.githubusercontent.com/26559841/107127997-6af88180-6877-11eb-8a31-4136a4e1017e.png)
https://discord.com/channels/122900397965705216/658550982975946762/804499734919774268

**For easier reference:**
If you are having errors when running `yarn build:production` (command is used to rebuild the Panel's UI after making changes) and are on `v1.2.2` or lower, you are probably missing the `.babel-plugin-macrosrc.js` file which is not included when installing the panel by following the documentation.  In order to fix those errors you will need to download the file, you can do this by running `wget https://raw.githubusercontent.com/pterodactyl/panel/v1.2.2/.babel-plugin-macrosrc.js`.

The two known errors this fixes is `Module not found: Error: Can't resolve '@emotion/react'` and `Error: .only must be an array, or undefined`, if your error is not one of those, this fix may or may not work.

**Comment by gOOvER** [comment]
He wanted to install an Addon. Addons are not suported atm. so ask Addon Author
